com.ticktick.TickTick.yml

```diff
@@ -0,0 +1,77 @@
+app-id: com.ticktick.TickTick
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: ticktick
+rename-desktop-file: ticktick.desktop
+rename-icon: ticktick
+
+tags:
+  - proprietary
+
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: ticktick
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra /app/bin/apply_extra
+      - install -Dm755 ticktick -t ${FLATPAK_DEST}/bin/
+      - install -Dm644 ticktick.desktop -t ${FLATPAK_DEST}/share/applications 
+      - desktop-file-edit --set-key=Exec --set-value=ticktick ${FLATPAK_DEST}/share/applications/ticktick.desktop
```

Comment:
since the .deskop is in the repository, you might as well make it right and withou the need to use this.